src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -5,8 +5,14 @@
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
 
 public class RegisterServletTest{
```

Comment:
Let's add a test for the positive case too (user does not exist yet, valid request comes in, user gets created).